torch/testing/_internal/composite_compliance.py

```diff
@@ -142,13 +142,19 @@ def __new__(cls, elem, *args, **kwargs):
                 device=elem.device, requires_grad=elem.requires_grad,
                 strides=elem.stride(), storage_offset=elem.storage_offset())
 
-            # CompositeCompliantTensor steals the "requires_grad"-ness.
             if elem.requires_grad:
-                # Why clone? Because sometimes OpInfo shares inputs between tests...
-                r.elem = elem.detach().clone()
```

Comment:
`clone` on non-contiguous tensor results in a contiguous tensor with different strides than original `elem`.